src/app/containers/Metadata/linkedDataAbout.js

```diff
@@ -0,0 +1,39 @@
+const acceptableTypes = ['Person', 'Event', 'Organization', 'Place'];
+
+export const checkType = types => {
+  if (types.length === 0 || types.length > 1) {
```

Comment:
What's the scenario for `types.length > 1` needing to return 'Thing'?

Reply:
Its when types doesn't have exactly one of the types mentioned in the acceptableTypes array